helm/chart/router/templates/deployment.yaml

```diff
@@ -99,10 +103,12 @@ spec:
             {{- toYaml .Values.resources | nindent 12 }}
           {{- if or .Values.router.configuration  .Values.extraVolumeMounts }}
           volumeMounts:
+            {{- if or .Values.router.configuration }}
             - name: router-configuration
```

Comment:
I don't think the `or` is required.

Reply:
yeah woops I just copy the previous line and that happened, still worked in testing tho... odd